django/views/csrf.py

```diff
@@ -131,4 +130,8 @@ def csrf_failure(request, reason=""):
         'docs_version': get_docs_version(),
         'more': _("More information is available with DEBUG=True."),
     })
+    try:
+        t = loader.get_template(template_name)
+    except TemplateDoesNotExist:
+        t = Engine().from_string(CSRF_FAILURE_TEMPLATE)
```

Comment:
Thanks for the feedback! I disagree that we set it to ``None`` hovewer, as the way I implemented it, it

* mirrors the behaviour of the ``not_found``, ``permission_denied``, … error views
* allows changing the error page by just dropping in a template file without needing to write a custom view at all